project-base/app/config/bundles.php

```diff
@@ -34,7 +34,7 @@
     Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true],
     Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true],
     Symfony\WebpackEncoreBundle\WebpackEncoreBundle::class => ['all' => true],
-    Shopsys\S3Bridge\S3BridgeBundle::class => ['all' => true],
+    Shopsys\S3Bridge\ShopsysS3BridgeBundle::class => ['all' => true],
```

Comment:
Should this be mentioned in UPGRADE or will Symfony update it on its own?

Reply:
true, added - fdad8c2ef8